exchange-deployer/src/lib.rs

```diff
@@ -0,0 +1,46 @@
+#![no_std]
+
+multiversx_sc::imports!();
+
+pub mod action_type;
+pub mod deploy_types;
+pub mod fee;
+
+#[multiversx_sc::contract]
+pub trait ExchangeDeployer:
```

Comment:
I think we should have an `only_owner` endpoint, that allows the owner of the exchange deployer to call any `only_owner` endpoint of the deployed contracts. Right now you can only deploy SCs and maybe whitelist the deployer as an admin. But in case we want to remove the deployer from the admin whitelist (for being malicious let's say), you cannot do so. And this extends to all kinds of use cases, where we want to call owner endpoints.